c2corg_ui/templates/waypoint/index.html

```diff
@@ -53,7 +54,6 @@
   </section>
 
   <div class="map-right">
-    ${toggle_map_btns()}
```

Comment:
If this helper function is no longer used:
* it should also be removed from the ``import`` at the top of the template https://github.com/c2corg/v6_ui/blob/e0a8e584a02a8679a7ab421c748a33680734bd59/c2corg_ui/templates/waypoint/index.html#L3 (ditto for other document types)
* the function should be removed from the helper file https://github.com/c2corg/v6_ui/blob/e0a8e584a02a8679a7ab421c748a33680734bd59/c2corg_ui/templates/helpers/list.html#L1-L12